client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -581,6 +588,26 @@ export class CheckoutThankYou extends Component<
 				pageContent = <DomainOnlyThankYou purchases={ purchases } receiptId={ receiptId } />;
 			} else if ( purchases.length === 1 && isPlan( purchases[ 0 ] ) ) {
 				pageContent = <PlanOnlyThankYou primaryPurchase={ purchases[ 0 ] } />;
+			} else if ( wasTitanEmailOnlyProduct ) {
+				const titanPurchase = purchases.find( ( purchase ) => isTitanMail( purchase ) );
+
+				pageContent = (
+					<TitanSetUpThankYou
+						domainName={ purchases[ 0 ].meta }
+						newQuantityOfMailboxes={ titanPurchase?.newQuantity }
```

Comment:
```suggestion
						numberOfMailboxesPurchased={ titanPurchase?.newQuantity }
```
I realize that the prop name was meant to match the endpoint property, but it doesn't quite provide a clear description of what the prop represents. I'd suggest renaming the prop to `numberOfMailboxesPurchased`.